CRM/Contact/Page/View.php

```diff
@@ -311,7 +311,13 @@ public static function setTitle($contactId, $isDeleted = FALSE) {
     }
     if ($isDeleted) {
       $title = "<del>{$title}</del>";
-      $mergedTo = civicrm_api3('Contact', 'getmergedto', ['contact_id' => $contactId, 'api.Contact.get' => ['return' => 'display_name']]);
+      try {
+        $mergedTo = civicrm_api3('Contact', 'getmergedto', ['contact_id' => $contactId, 'api.Contact.get' => ['return' => 'display_name']]);
+      }
+      catch (Exception $e) {
```

Comment:
Can we catch the more specific CiviCRM_Api_Exception?

Reply:
done